README.md

```diff
@@ -34,102 +31,6 @@ make -j4
-````
-echo 'export GAZEBO_MODEL_PATH=~/ardupilot_gazebo/models' >> ~/.bashrc
-````
-
-Set Path of Gazebo Worlds (Adapt the path to where to clone the repo)
-````
-echo 'export GAZEBO_RESOURCE_PATH=~/ardupilot_gazebo/worlds:${GAZEBO_RESOURCE_PATH}' >> ~/.bashrc
-````
-
-````
-source ~/.bashrc
-````
-
-DONE !
-
-Now launch a world file with a copter/rover/plane and ardupilot plugin, and it should work! 
-(I will try to add some world file and model later)
-
-## HELP
-
-How to Launch :  
-Launch Ardupilot Software In the Loop Simulation for each vehicle.
-On new terminal, launch Gazebo with basic demo world.
-
-#####ROVER (no model provided for now)
-
-On 1st Terminal (Launch Ardupilot SITL)
-````
-sim_vehicle.py -v APMrover2 -f gazebo-rover --map --console
-````
-
-On 2nd Terminal (Launch Gazebo with demo Rover model)
-````
-gazebo --verbose worlds/ (Please Add if there is one.)
-````
-
-##### COPTER
-
-On 1st Terminal (Launch Ardupilot SITL)
-````
-sim_vehicle.py -v ArduCopter -f gazebo-iris --map --console
-````
-
-On 2nd Terminal (Launch Gazebo with demo 3DR Iris model)
-````
-gazebo --verbose worlds/iris_arducopter_runway.world
-````
-
-##### PLANE
-
-On 1st Terminal (Launch Ardupilot SITL)
-````
-sim_vehicle.py -v ArduPlane -f gazebo-zephyr --map --console
-````
-
-On 2nd Terminal (Launch Gazebo with demo Zephyr flying wing model)
-````
-gazebo --verbose worlds/zephyr_ardupilot_demo.world
-````
-
-In addition, you can use any GCS of Ardupilot locally or remotely (will require connection setup).
-If MAVProxy Developer GCS is uncomportable. Omit --map --console arguments out of SITL launch and use APMPlanner 2 or QGroundControl instead.
-Local connection with APMPlanner2/QGroundControl is automatic, and recommended.
-
 ## Troubleshooting
 
-### Missing libArduPilotPlugin.so... etc 
-
-In case you see this message when you launch gazebo with demo worlds, check you have no error after sudo make install.  
-If no error use "ls" on the install path given to see if the plugin is really here.  
-If this is correct, check with `cat /usr/share/gazebo/setup.sh` the variable `GAZEBO_PLUGIN_PATH`. It should be the same as the install path. If not use `cp` to copy the lib to right path. 
-
-For Example
-
-````
-sudo cp -a /usr/lib/x86_64-linux-gnu/gazebo-7.0/plugins/ /usr/lib/x86_64-linux-gnu/gazebo-7/
-````
-
-path mismatch is confirmed as ROS's glitch. It will be fixed.
-
-### Future(not activated yet)
-To use Gazebo gps, you must offset the heading of +90° as gazebo gps is NWU and ardupilot is NED 
-(I don't use GPS altitude for now)  
-example : for SITL default location
-````
-    <spherical_coordinates>
-      <surface_model>EARTH_WGS84</surface_model>
-      <latitude_deg>-35.363261</latitude_deg>
-      <longitude_deg>149.165230</longitude_deg>
-      <elevation>584</elevation>
-      <heading_deg>87</heading_deg>
-    </spherical_coordinates>
-````
-Rangefinder
+See https://ignitionrobotics.org/docs/edifice/troubleshooting#ubuntu
```

Comment:
We should add a license and credits section. The current license file is GPL3 which is consistent with the rest of ArduPilot. I'm in favour of that, but tridge and the other developers may have a view on what's best.

Regarding credits: the initial port to Ignition (non-JSON version) was carried out by the OSRF team and can be referenced here: https://ignitionrobotics.org/api/gazebo/6.4/ardupilot.html

Reply:
I have made the original plugin GPL, but I am pretty sure it cannot be GPL due to linkage issue with gazebo. Therefore maybe a LGPLv3 would fit.

Reply:
@khancyr I'm not sure about the fine details of when a library can / can't be GPL. Is the thinking that because the plugin links to Ignition that there is some transitive requirement that Ignition also be GPL (which it's not and won't be)?

It would surprise me if a downstream library (the plugin) could force an upstream licensing requirement (Ignition), but I really don't know the specifics - so will leave that one to the dev-team's lawyers - but I'd be interested to know the argument for / against what ever way it goes.